lib/integer.gd

```diff
@@ -990,6 +990,7 @@ DeclareGlobalFunction( "SmallestRootInt" );
 ##  <Description>
 ##  prints the prime factorization of the integer <A>n</A> in human-readable
 ##  form.
+##  See also <Ref Func="StringPP"/>.
```

Comment:
Actually, it turns out I misremembered the bit about differences between StringPP and PrintFactorsInt. So I rewrote the latter to use the former